### PR TITLE
check if feed item is a club post

### LINKED ISSUE #13: Issue retrieving own activity's owner name
Looking over the logs, it seems like this line is having trouble.
```python
 h = container.get_by_test_id("owners-name").get_attribute('href')
```
The id might not exist anymore, or under a different structure. Would be good to have a fix.

### PR DESCRIPTION
**Pull request by domantascibas**
resolves #13 caused by club posts in the feed

### CODE REVIEW

File: give_kudos.py [hunk 1]
```diff
@@ -102,9 +107,23 @@ def locate_kudos_buttons_and_maybe_give_kudos(self, web_feed_entry_locator) -> i
...
+    def is_club_post(self, container) -> bool:
+        """
+        Returns true if the container is a club post
+        """
+        print("group header:", container.get_by_test_id("group-header").count())
+        if(container.get_by_test_id("group-header").count() > 0):
+            return True
+        
+        print("club member post:", container.locator(".clubMemberPostHeaderLinks").count())
+        if(container.locator(".clubMemberPostHeaderLinks").count() > 0):
```
Comment (isaac-chung) [inline give_kudos.py:122]: Remove these print statements?

**Review by isaac-chung — COMMENTED** [review]
Looks great, thank you! Just wanted to make sure these were for debugging.

File: give_kudos.py (hunk 1 above)
Comment (domantascibas) [reply give_kudos.py:122]: good catch! I removed other debug lines, but these two slipped trough 😄

Comment (domantascibas) [reply give_kudos.py:122]: yes, need to remove these two print statements

**Comment by domantascibas** [comment]
just pushed an update

**Pull request merged**